### DISCUSSION TITLE
Helmet change

### DISCUSSION THREAD

**Discussion by TechTonic02** [Malcolm-Q/LC-LateGameUpgrades#443] [Ideas]
Basically instead of completely negating damage for a certain amount of hits, it would instead reduce damage by a certain amount and when receiving damage that would kill you, it would instead reduce your health to 1 and destroy the helmet, (and possibly add the option to configurate them)

**Comment by WhiteSpike** [comment]
In the case of one shots (dog lunge or bracken snap), it would still do the same logic?

**Reply by WhiteSpike** [reply]
Anyways, I implemented what you suggested, basically:
- You can pick two modes of damage mitigation:
  - TotalPerHit, where it takes hits from the helmet and completely mitigates damage and breaks when you have reached the set amount of hits
  - PartialTilLowHealth, where it partially mitigates damage and only breaks when the next hit the player would have received would end up in their death.
  
This should be close to what you suggested.

**Reply by TechTonic02** [reply]
Oh I haven't check in for awhile, but yeah that's exactly what I was thinking, thanks for implementing it, I considered the helmet to either be too weak or too strong depending on the amount of hits, so thanks again!
